specs/merge/beacon-chain.md

```diff
@@ -68,7 +68,7 @@ Warning: this configuration is not definitive.
 | Name | Value |
 | - | - |
 | `MERGE_FORK_VERSION` | `Version('0x02000000')` |
-| `MERGE_FORK_EPOCH` | `Epoch(18446744073709551615)` **TBD** |
+| `MERGE_FORK_EPOCH` | `Epoch(9223372036854775808)` **TBD** |
```

Comment:
the merge and sharding constants have to change as they are downstream of the altair fork so they need to have the alignment condition as well?

Reply:
That was only for alignment. Open to discussion! :)